app/controllers/v2/actions_controller.rb

```diff
@@ -1,5 +1,15 @@
 module V2
   class ActionsController < ApplicationController
+    def index
+      TimedFeature.check!(owner: "Tijmen", expires: "2017-04-22")
+
+      actions = Action
+        .where(content_id: params[:content_id])
+        .as_json(only: %i[action user_uid created_at])
```

Comment:
Could we add a limit here? pagination would be better of course, but understand it's pretty bare bones